x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/edit_connector_flyout/index.tsx

```diff
@@ -131,6 +131,7 @@ const EditConnectorFlyoutComponent: React.FC<EditConnectorFlyoutProps> = ({
   const { preSubmitValidator, submit, isValid: isFormValid, isSubmitting } = formState;
   const hasErrors = isFormValid === false;
   const isSaving = isUpdatingConnector || isSubmitting || isExecutingConnector;
+  const [currentConnector, setCurrentConnector] = useState<ActionConnector>(connector);
```

Comment:
nit: maybe rename this to `savedConnector` or `updatedConnector` you might get naming conflicts with the latter though.

Reply:
Moved updated connector logic in `x-pack/plugins/cases/public/components/configure_cases/index.tsx`